app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/FinishAndSignOut.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.fhregistrationfrontend.forms.navigation.Navigation
+@import uk.gov.hmrc.fhregistrationfrontend.forms.navigation._
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@(navigation: Navigation, viewHelpers: ViewHelpers)(implicit messages: Messages)
+
+@backLink = @{
```

Comment:
If this is same as what is there in Back.scala.html, you can directly call that here like @Back(navigation, viewhelpers)

Reply:
Not sure why you can see that page - it was deleted on my side.  Please ignore

Reply:
If this can be seen in the PR, this will go in merge pls remove it . it can't be ignored